mozilla/donate-wagtail
# Send data to basket before sending to SQS

Closes #1394 

Steps to test:
1. Make sure your braintree env vars are set. Ask me or Daniel for them if you need sandbox settings. 
2. Make a donation. 
3. On page 3 there's a "thank you" page. It requires a donation to be made before being able to view it. 
4. Add your email address, check the checkbox, and subscribe. We should see no errors. 

Env vars already set for Thunderbird (RA and Production) along with the main donate site (RA and Production) 

Review comments below.
